eos-tech-support/eos-convert-system

```diff
@@ -5,6 +5,11 @@ if [ ! -L /ostree ]; then
   exit 1
 fi
 
+if ! eos-check-secureboot; then
+  echo "Please reboot with secure boot disabled" >&2
```

Comment:
Please make this a little clearer as a failure, right now it could be interpreted that eos-convert-system has done something and all you have to do is reboot. (when in fact you have to run eos-convert-system again)